apps/backoffice-v2/src/domains/workflows/fetchers.ts

```diff
@@ -54,20 +57,35 @@ export const fetchWorkflows = async (params: {
 
 export type TWorkflowById = z.output<typeof WorkflowByIdSchema>;
 
+export const BaseWorkflowDefinition = ObjectWithIdSchema.extend({
```

Comment:
Usage does not seem to require two schemas.
```suggestion
export const WorkflowDefinition = ObjectWithIdSchema.extend({
```